test-package-compiler/src/generated_tests.rs

```diff
@@ -183,144 +183,144 @@ fn imported_external_fns() {
+        crate::prepare("./cases/opaque_type_construct");
     insta::assert_snapshot!(
-        "unknown_module_field_in_expression",
+        "opaque_type_construct",
         output,
-        "./cases/unknown_module_field_in_expression"
+        "./cases/opaque_type_construct"
     );
 }
 
 #[rustfmt::skip]
 #[test]
-fn javascript_import() {
+fn opaque_type_destructure() {
     let output =
-        crate::prepare("./cases/javascript_import");
+        crate::prepare("./cases/opaque_type_destructure");
     insta::assert_snapshot!(
-        "javascript_import",
+        "opaque_type_destructure",
         output,
-        "./cases/javascript_import"
+        "./cases/opaque_type_destructure"
     );
 }
 
 #[rustfmt::skip]
 #[test]
-fn variable_or_module() {
+fn src_importing_test() {
     let output =
-        crate::prepare("./cases/variable_or_module");
+        crate::prepare("./cases/src_importing_test");
     insta::assert_snapshot!(
-        "variable_or_module",
+        "src_importing_test",
         output,
-        "./cases/variable_or_module"
+        "./cases/src_importing_test"
     );
 }
 
 #[rustfmt::skip]
 #[test]
-fn imported_constants() {
+fn unknown_module_field_in_constant() {
     let output =
-        crate::prepare("./cases/imported_constants");
+        crate::prepare("./cases/unknown_module_field_in_constant");
     insta::assert_snapshot!(
-        "imported_constants",
+        "unknown_module_field_in_constant",
         output,
-        "./cases/imported_constants"
+        "./cases/unknown_module_field_in_constant"
     );
 }
 
 #[rustfmt::skip]
 #[test]
-fn erlang_import_shadowing_prelude() {
+fn unknown_module_field_in_expression() {
     let output =
-        crate::prepare("./cases/erlang_import_shadowing_prelude");
+        crate::prepare("./cases/unknown_module_field_in_expression");
     insta::assert_snapshot!(
-        "erlang_import_shadowing_prelude",
+        "unknown_module_field_in_expression",
         output,
-        "./cases/erlang_import_shadowing_prelude"
+        "./cases/unknown_module_field_in_expression"
     );
 }
 
 #[rustfmt::skip]
 #[test]
-fn javascript_empty() {
+fn unknown_module_field_in_import() {
     let output =
-        crate::prepare("./cases/javascript_empty");
+        crate::prepare("./cases/unknown_module_field_in_import");
     insta::assert_snapshot!(
-        "javascript_empty",
+        "unknown_module_field_in_import",
         output,
-        "./cases/javascript_empty"
+        "./cases/unknown_module_field_in_import"
     );
 }
 
 #[rustfmt::skip]
 #[test]
-fn opaque_type_construct() {
+fn variable_or_module() {
     let output =
-        crate::prepare("./cases/opaque_type_construct");
+        crate::prepare("./cases/variable_or_module");
     insta::assert_snapshot!(
-        "opaque_type_construct",
+        "variable_or_module",
         output,
-        "./cases/opaque_type_construct"
+        "./cases/variable_or_module"
```

Comment:
Looks like I need to sort this or something to make the order stable! Oops my bad